aviatrix/resource_aviatrix_spoke_gateway.go

```diff
@@ -294,6 +295,40 @@ func resourceAviatrixSpokeGateway() *schema.Resource {
 				ForceNew:    true,
 				Description: "Enable BGP. Default: false.",
 			},
+			"bgp_ecmp": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     false,
+				Description: "Enable Equal Cost Multi Path (ECMP) routing for the next hop.",
+			},
+			"enable_active_standby": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     false,
+				Description: "Enables Active-Standby Mode, available only with HA enabled.",
+			},
+			"prepend_as_path": {
+				Type:        schema.TypeList,
+				Optional:    true,
```

Comment:
If this is a list, is it possible that backend return something in a different order, causing a diff?

Reply:
prepend_as_path is a list of repeated as num for the gateway so order shouldn't matter

Reply:
Correct, it does not matter.